extensions/amp-font/0.1/amp-font.js

```diff
@@ -124,7 +128,7 @@ export class AmpFont extends AMP.BaseElement {
       this.onFontLoadSuccess_();
     }).catch(error => {
       this.onFontLoadError_();
-      throw error;
+      log.warn(TAG_, error, this.element);
```

Comment:
The logger isn't the right tool in its current form as content devs never see it.

@erwinmombay was looking into fixing that, but I believe for now we should use console.warn/log.